src/app.sass

```diff
@@ -23,8 +23,12 @@ html, body, .app
 html
 	box-sizing: border-box
 
-.text-center
-	text-align: center
+.text
+	&-center
+		text-align: center
+
+	&-left
+		text-align: left !important
```

Comment:
This can surely be solved without `!important`

Reply:
done